### DISCUSSION TITLE
Mesh graphormer hand refiner

### DISCUSSION THREAD

**Discussion by monstamash77** [lllyasviel/Fooocus#2542] [Ideas]
Any plans to add mesh graphormer hand refiner to fooocus? I prefer working in fooocus to comfyui, but hands has always been an issue. This workflow does a great job of fixing them

**Comment by aolko** [comment]
please include a source url

**Comment by monstamash77** [comment]
Sure, this is the model used by mesh gramographer is comfy: https://github.com/wenquanlu/HandRefiner
and example workflow: https://openart.ai/workflows/oliviosarikas/mesh-graphormer-hand-fixing-controlnet/NkhzwEW80FzCcvzzXEsH

**Reply by mashb1t** [reply]
relates to https://github.com/lllyasviel/Fooocus/issues/1831, currently in the backlog
